core/src/main/java/com/google/zxing/qrcode/encoder/QRCode.java

```diff
@@ -38,6 +38,9 @@ public QRCode() {
     maskPattern = -1;
   }
 
+  /**
+   * @return the mode. Not relevant if {@link com.google.zxing.EncodeHintType.java#QR_COMPACT} is selected.
```

Comment:
Why isn't it relevant - it still selects some mode no?

Reply:
Yes, it sets it to a constant value (currently Mode.BYTE). I could make it return the value of the first mode value in the bits from the left. My thinking was, that asking for "the" mode in a multi mode code doesn't make much sense so that added that comment.